helm/linkerd2-multicluster-app/README.md

```diff
@@ -0,0 +1,94 @@
+# linkerd-multicluster
+
+The Linkerd-Multicluster extension contains resources to support multicluster
+linking to remote clusters
+
+![Version: 30.1.1-edge](https://img.shields.io/badge/Version-30.1.1--edge-informational?style=flat-square)
+
+![AppVersion: edge-XX.X.X](https://img.shields.io/badge/AppVersion-edge--XX.X.X-informational?style=flat-square)
+
+**Homepage:** <https://linkerd.io>
+
+## Quickstart and documentation
+
+You can run Linkerd on any Kubernetes 1.20+ cluster in a matter of seconds. See
+the [Linkerd Getting Started Guide][getting-started] for how.
```

Comment:
Shouldn't we reference our linkerd2-app here?

Reply:
good point